socorro/processor/signature_utilities.py

```diff
@@ -683,6 +685,10 @@ def _create_frame_list(
         return frame_signatures_list
 
     #--------------------------------------------------------------------------
+    def _get_crashing_thread(self, processed_crash):
+        return processed_crash.json_dump['crash_info']['crashing_thread']
```

Comment:
We had to do a lot of magic on the webapp because some json dumps were spuriously missing things like this `crashing_thread`. Should we be more careful here?